server/src/scheduling/scheduler.js

```diff
@@ -106,7 +100,7 @@ const getOptimalRoute = async (appointments) => {
   const data = await resp.json();
 
   if (!resp.ok) {
-    throw new Error("Couldn't connect to graph hopper");
+    throw new Error(`Couldn't connect to graph hopper: ${response.status}`);
```

Comment:
```suggestion
    throw new Error(`Couldn't connect to graph hopper: ${resp.status}`);
```
`response` doesn't exist